### PR TITLE
add Santa web component to storybook

### PR DESCRIPTION
**Pull request by daniel8lopes**

### CODE REVIEW

File: webcomponents/web-components/Santa/santa.stories.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { storiesOf } from '@storybook/html';
+
+// import component
+import './santa';
+import readme from './readme.md';
+
+storiesOf('Data Input|Select box', module)
+    .add('default', () => {
+        const component = createElement();
+
```
Comment (gsferreira) [inline webcomponents/web-components/Santa/santa.stories.js]: clear space

File: webcomponents/web-components/Santa/santa.stories.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { document, console } from 'global';
+import { storiesOf } from '@storybook/html';
+
+// import component
+import './santa';
+import readme from './readme.md';
+
+storiesOf('Data Input|Select box', module)
```
Comment (gsferreira) [inline webcomponents/web-components/Santa/santa.stories.js]: this name doesn't make sense

File: webcomponents/web-components/Santa/readme.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[View source code](https://github.com/OMNIALowCode/omnia3-samples/blob/master/webcomponents/web-components/Santa/santa.js)
+
+
+### Attribution
+
+|CSS Santa was made by @alireza29675, do follow her on Twitter|
```
Comment (gsferreira) [inline webcomponents/web-components/Santa/readme.md]: Add link to codepen

File: webcomponents/web-components/Santa/santa.stories.js (hunk 1 above)
Comment (nunofilipesf) [inline webcomponents/web-components/Santa/santa.stories.js]: Remove the blank lines

File: webcomponents/web-components/Santa/santa.stories.js (hunk 2 above)
Comment (nunofilipesf) [inline webcomponents/web-components/Santa/santa.stories.js]: Review the Story name

File: webcomponents/web-components/Santa/readme.md (hunk 3 above)
Comment (nunofilipesf) [inline webcomponents/web-components/Santa/readme.md]: "her"?

File: webcomponents/web-components/Santa/santa.js [hunk 4]
```diff
@@ -0,0 +1,827 @@
...
+  .santa-wrapper {
+	  display: inline-block;
+	  background: red;
+	  width: 100%;
+	  height: 400px;
+	  border-radius: 0px;
+	  -webkit-box-shadow: 0px 0px 10px 0px rgba(0,105,11,1);
+	  -moz-box-shadow: 0px 0px 10px 0px rgba(0,105,11,1);
+	  box-shadow: 0px 0px 10px 0px rgba(0,105,11,1);
+	  //border: solid 10px white;
```
Comment (nunofilipesf) [inline webcomponents/web-components/Santa/santa.js]: Remove commented code

File: webcomponents/web-components/Santa/santa.js [hunk 5]
```diff
@@ -0,0 +1,827 @@
...
+	18% {
+	  bottom: -30px; }
+  
+	20% {
+	  bottom: -28px; }
+  
+	22% {
+	  bottom: -30px; }
+  
+	24% {
```
Comment (nunofilipesf) [inline webcomponents/web-components/Santa/santa.js:583]: Please format the code

**Review by nunofilipesf — CHANGES_REQUESTED** [review]

**Comment by nunofilipesf** [comment]
@daniel8lopes the build files are missing

File: webcomponents/web-components/Santa/santa.stories.js (hunk 2 above)
Comment (gsferreira) [reply webcomponents/web-components/Santa/santa.stories.js]: To be more clear, we are saying this because this component isn't a Select box.

Comment (daniel8lopes) [reply webcomponents/web-components/Santa/santa.stories.js]: fixed

Comment (daniel8lopes) [reply webcomponents/web-components/Santa/santa.stories.js]: thanks, it should be fine now

File: webcomponents/web-components/Santa/santa.js (hunk 5 above)
Comment (daniel8lopes) [reply webcomponents/web-components/Santa/santa.js:583]: done

File: webcomponents/web-components/Santa/readme.md (hunk 3 above)
Comment (daniel8lopes) [reply webcomponents/web-components/Santa/readme.md]: done

File: webcomponents/web-components/Santa/santa.js (hunk 4 above)
Comment (daniel8lopes) [reply webcomponents/web-components/Santa/santa.js]: done

File: webcomponents/web-components/Santa/santa.stories.js (hunk 1 above)
Comment (daniel8lopes) [reply webcomponents/web-components/Santa/santa.stories.js]: done

File: webcomponents/web-components/Santa/readme.md (hunk 3 above)
Comment (gsferreira) [reply webcomponents/web-components/Santa/readme.md]: Where is the link?

File: webcomponents/web-components/Santa/santa.stories.js [hunk 6]
```diff
@@ -5,18 +5,13 @@ import { storiesOf } from '@storybook/html';
 import './santa';
 import readme from './readme.md';
 
-storiesOf('Data Input|Select box', module)
+storiesOf('MerryChristmas', module)
```
Comment (nunofilipesf) [inline webcomponents/web-components/Santa/santa.stories.js]: Change to "Merry Christmas" (with a space between words)

**Comment by nunofilipesf** [comment]
@daniel8lopes  The build files are still missing

**Review by nunofilipesf — CHANGES_REQUESTED** [review]

**Review by nunofilipesf — APPROVED** [review]

**Pull request merged**